### PR TITLE
change studio logout urls and add namespace for colaraz_features

### PR DESCRIPTION
**Pull request by HamzaIbnFarooq**
### Story Link
https://edlyio.atlassian.net/browse/EDE-427
### Description
Changing the logout process on studio. Now the logout process will be governed by IdP with respect to the site-specific domains.

**Note**: Now we have to add 'LMS_BASE' and 'CMS_BASE' in site configurations of each site.

### CODE REVIEW

File: cms/templates/widgets/user_dropdown.html [hunk 1]
```diff
@@ -5,6 +5,9 @@
   from django.urls import reverse
   from django.utils.translation import ugettext as _
   from student.roles import GlobalStaff
+  from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
```
Comment (imhassantariq) [inline cms/templates/widgets/user_dropdown.html:8]: Why we are not implementing this file change in the theme?

File: cms/templates/widgets/user_dropdown.html [hunk 2]
```diff
@@ -5,6 +5,9 @@
...
+
+  lms_logout_url = configuration_helpers.get_value('LMS_BASE') + reverse('colaraz_features:auth_logout_redirect')
```
Comment (danialmalik) [inline cms/templates/widgets/user_dropdown.html:10]: Use `format` for string concatenation ?

File: lms/templates/courseware/course_about.html [hunk 3]
```diff
@@ -62,7 +62,7 @@
                         'You may need to {start_logout_tag}log out{end_tag} then try the enroll button again. '
                         'Please visit the {start_help_tag}help page{end_tag} for a possible solution.')).format(
                           start_help_tag=HTML("<a href='{url}'>").format(url=marketing_link('FAQ')), end_tag=HTML('</a>'),
-                          start_logout_tag=HTML("<a href='{url}'>").format(url=reverse('auth_logout_redirect'))
+                          start_logout_tag=HTML("<a href='{url}'>").format(url=reverse('colaraz_features:auth_logout_redirect'_logout_redirect'))
```
Comment (danialmalik) [inline lms/templates/courseware/course_about.html]:
```suggestion
                          start_logout_tag=HTML("<a href='{url}'>").format(url=reverse('colaraz_features:auth_logout_redirect'))
```
I think it was left here by mistake

Comment (HamzaIbnFarooq) [reply lms/templates/courseware/course_about.html]: Already update.

File: cms/templates/widgets/user_dropdown.html (hunk 2 above)
Comment (HamzaIbnFarooq) [reply cms/templates/widgets/user_dropdown.html:10]: does it make any difference?

Comment (danialmalik) [reply cms/templates/widgets/user_dropdown.html:10]: depends if consistency is required or not

Comment (HamzaIbnFarooq) [reply cms/templates/widgets/user_dropdown.html:10]: basically, according to my understanding, format() is not solely for string concentrations. It is used when we have some hardcoded strings in it, but in this scenario its just two variables.

File: cms/templates/widgets/user_dropdown.html (hunk 1 above)
Comment (HamzaIbnFarooq) [reply cms/templates/widgets/user_dropdown.html:8]: Good catch, Currently we don't have any theme for studio. and as far as the urls of lms are concerned I have to change this PR.

Comment (HamzaIbnFarooq) [reply cms/templates/widgets/user_dropdown.html]: Added a todo. Will update this in theme as well when the theme will be in working condition.

**Pull request merged**